hardware/arduino/avr/libraries/EEPROM/README.md

```diff
@@ -66,7 +66,7 @@ This function returns a reference to the `object` passed in. It does not need to
 
 #### **Subscript operator: `EEPROM[address]`** [[_example_]](examples/eeprom_crc/eeprom_crc.ino)
 
-This operator allows using the identifier `EEPROM` like an array.  
```

Comment:
This trailing whitespace was intentional but I don't think the readability is significantly harmed by its removal.